docker-compose.yml

```diff
@@ -122,6 +122,28 @@ services:
     labels:
       logs: "promtail"
 
+    web-client:
+    # Configuration for the web client service
+    image: <web_client_image>:<tag>  
```

Comment:
As it stands, we don't have Docker images for the web client on Docker Hub, we'll have to build and start the service instead.